dashboards/cockroach-db/cockroach-db.json

```diff
@@ -68,7 +68,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "SELECT  sum(`Max_capacity`) As 'Max Capacity', ((average(`Max_capacity`) * cardinality(`Max_capacity`)) - (average(`Available_capacity`) * cardinality(`Available_capacity`))) As 'Used Capacity' ,  sum(`Available_capacity`) As 'Avaliable Capacity' FROM (SELECT (average(capacity) * cardinality(capacity)) AS `Max_capacity`, (average(capacity_available) * cardinality(capacity_available)) AS `Available_capacity` FROM Metric  WHERE ((app_kubernetes_io_name LIKE '%cockroach%' or app_newrelic_com_name LIKE '%cockroach%') AND (instrumentation.name='remote-write') )  FACET instance ) FACET instance"
+                "query": "SELECT  sum(`Max_capacity`) As 'Max Capacity', ((average(`Max_capacity`) * cardinality(`Max_capacity`)) - (average(`Available_capacity`) * cardinality(`Available_capacity`))) As 'Used Capacity' ,  sum(`Available_capacity`) As 'Avaliable Capacity' FROM (SELECT (average(capacity) * cardinality(capacity)) AS `Max_capacity`, (average(capacity_available) * cardinality(capacity_available)) AS `Available_capacity` FROM Metric  WHERE  (app_kubernetes_io_name LIKE '%cockroach%' or app_newrelic_com_name LIKE '%cockroach%' or k8s_app LIKE '%cockroach%') AND (instrumentation.name = 'remote-write') AND (prometheus_server IN ({{prometheus_server}})) AND (instance IN ({{instance}})) FACET ({{dimensions}}) ) FACET ({{dimensions}})"
```

Comment:
```
FACET ({{dimensions}}) ) FACET ({{dimensions}})
```

2 facets on this condition, please remove 1

Reply:
This is nested query and facets are part of respective queries and not for same queries. both are needed and same have been tested. 
Making no changes here.